proto/blob/events.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
```

Comment:
[blocking] Based on the following, I would expect the filename to be `event.proto` and not `events.proto`:

> Typed Events are Protobuf-defined [messages](https://docs.cosmos.network/main/architecture/adr-032-typed-events) used by the Cosmos SDK for emitting and querying Events. They are defined in a event.proto file, on a per-module basis and are read as proto.Message. Legacy Events are defined on a per-module basis in the module's /types/events.go file. They are triggered from the module's Protobuf [Msg service](https://docs.cosmos.network/main/building-modules/msg-services) by using the [EventManager](https://docs.cosmos.network/main/core/events#eventmanager).

Ref: https://docs.cosmos.network/main/core/events